app/views/posts/show.html.erb

```diff
@@ -21,4 +25,11 @@
       <% end %>
     </div>
   </div>
+
+  <%= form_with model: @comment, url:  user_post_comments_path(user_id: @current_user.id, post_id: @post.id), method: :post, scope: :comment do |f| %>
+  <div>
+    <%= f.text_area :text, placeholder: "Add Comment" %>
+    <%= f.submit "Comment" %>
+  </div>
+  <% end %>
```

Comment:
- [OPTIONAL] Great work man. I love the way you structured your code. I would advise you to make the `Comments` form a partial. That way, you can render it wherever you need it and you will be applying the **_Single Responsibility Rule_**. More like the concept of components in react 👍 😎